zipline/finance/performance/period.py

```diff
@@ -193,15 +227,14 @@ def position_tracker(self, obj):
         # we only calculate perf once we inject PositionTracker
         self.calculate_performance()
 
-    def rollover(self):
-        self.starting_value = self.ending_value
-        self.starting_exposure = self.ending_exposure
-        self.starting_cash = self.ending_cash
-        self.period_cash_flow = 0.0
-        self.pnl = 0.0
-        self.processed_transactions = {}
-        self.orders_by_modified = {}
-        self.orders_by_id = OrderedDict()
+    def rollover(self, capital_change=0):
```

Comment:
Ok, becoming clearer now. (`initialize` and `subdivide` setting many of the same values was blurring that distinction.)

In that case, is it necessary to call `rollover` in `process_capital_changes` since `handle_daily_close` will also call `rollover`? (If the change is made to pull `self.ending_cash += capital_change` into `process_capital_changes`?) I don't think it is accurate to do 

Some other proposed changes:
- Make the "unset" values of `_prev_subperiod_pnl` etc. in `initialize` to be `None`. It is not that there was previous period with returns of 0, but that there was no previous period. (Am I understanding that correctly?)
- We may want to move `rollover` out of the shared `PerformancePeriod` class and make a `DailyPerformancePeriod` which clearly shows that rollover is only used in that context. (Would not block PR on breaking out 'rollover'.)